features/bootstrap/FeatureContext.php

```diff
@@ -355,6 +421,66 @@ public function thereIsDummyObjectsWithDummyBoolean($nb, $bool)
     }
 
     /**
+     * @Given there is :nb embedded dummy objects with embeddedDummy.dummyBoolean :bool
+     */
+    public function thereIsDummyObjectsWithEmbeddedDummyBoolean($nb, $bool)
+    {
+        if (in_array($bool, ['true', '1', 1], true)) {
```

Comment:
Hmm, I don't see what can block here. We're not talking about a filter here but a setter. It's just a step definition with an argument which is converted to a pure boolean in all cases! So why couldn't we use `FILTER_VALIDATE_BOOLEAN`?🤔